<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="src_label">
        <property name="text">
         <string>Source dir:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="src_edit"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="browse_src_btn">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dst_label">
        <property name="text">
         <string>Target dir:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dst_edit"/>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="browse_dst_btn">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="delete_processed_chb">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete processed files from source directory (files will be &lt;span style=&quot; font-weight:600;&quot;&gt;moved&lt;/span&gt; to target directory).&lt;/p&gt;&lt;p&gt;Source directory will be removed if there is no more files in it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Delete processed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fix_extensions_chb">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change file extension to correct if incorrect extension will detected.&lt;/p&gt;&lt;p&gt;Unsupported files will not changed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Fix extensions</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_stop_btn">
        <property name="text">
         <string>GO!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="log_view">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>src_edit</tabstop>
  <tabstop>browse_src_btn</tabstop>
  <tabstop>dst_edit</tabstop>
  <tabstop>browse_dst_btn</tabstop>
  <tabstop>delete_processed_chb</tabstop>
  <tabstop>fix_extensions_chb</tabstop>
  <tabstop>start_stop_btn</tabstop>
  <tabstop>log_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
